src/uvw/util.hpp

```diff
@@ -11,6 +11,12 @@
 #include <array>
 #include <uv.h>
 
+#ifdef _WIN32
```

Comment:
I get the following error on my machine with or without the workaround mentioned below:
```
t:\gitrepo\uvw\src\uvw\util.hpp(107): error C3490: 'flags' cannot be modified because it is being accessed through a const object [T:\GitRepo\uvw\build\test\odr.vcxproj]
t:\gitrepo\uvw\src\uvw\util.hpp(107): error C2440: 'return': cannot convert from 'const uvw::Flags<uvw::ProcessHandle::Process>' to 'uvw::Flags<uvw::ProcessHandle::Process> &' [T:\GitRepo\uvw\build\test\odr.vcxproj]
```
Where the lines 106-108 in `util.hpp` are the ones taken from the example.
I'm certain that the same will happen on AppVeyor machines as well. Please tell me if a test is needed.